style.css

```diff
@@ -559,3 +566,241 @@ li a {
+    flex-direction: row;
+  }
+
+  .hide {
+    display: block;
+  }
+
+  .aaa {
+    background-image: url(./image/prof-print.png);
+    margin-bottom: 50px;
+  }
+
+  .bbb {
+    background-image: url(./image/dash-data.png);
+  }
+
+  .ccc {
+    background-image: url(./image/websitepro.png);
+  }
+
+  .content-container {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .bb1 {
+    display: none;
+  }
+
+  .aa1 {
+    width: 371px;
+  }
+
+  .menuitems {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    left: 143px;
+  }
+
+  .main-section {
+    background-color: #1c1a19;
+    height: 670px;
+    background-image: url(image/sidepic1.png);
+    background-position: 100% 100%;
+    background-repeat: no-repeat;
+    background-size: 500px;
+  }
+
+  .icons {
+    justify-content: space-between;
+    list-style-type: none;
+    padding: 0 20px 0 20px;
+    position: absolute;
+  }
+
+  .online-icons {
+    display: inline-block;
+    justify-content: flex-start;
+    list-style-type: none;
+    margin-top: 22%;
+    position: absolute;
+  }
+
+  .first-box {
+    flex-direction: row;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-bottom: 25px;
+  }
+
+  .firstcard {
+    margin-bottom: 20px;
+  }
+
+  .line111,
+  .line22 {
+    display: none;
+  }
+
+  .aboutme-c {
+    margin-top: 6rem;
+    background-image: url(./image/sidepic22.png);
+    background-repeat: no-repeat;
+    background-size: 450px;
+    background-position: 100% 38%;
+  }
+
+  .langs {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 50px 0 0 0;
+  }
+
+  .a1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 170px;
+    padding-top: 0;
+    width: 289.46px;
+    margin-top: 0;
+  }
+
+  .a2 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-left: 200px;
+  }
+
+  .a3 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding-left: 200px;
+  }
+
+  .about-me {
+    height: 1250px;
+  }
+
+  .form-section {
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 130px 0;
+  }
+
+  .formtext {
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+  }
+
+  .iconzz {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+  }
+
+  .formtext p {
+    font-family: inter, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 32px;
+    text-align: left;
+    padding-left: 70px;
+  }
+
+  .main-form {
+    display: none;
+  }
+
+  .form1 {
+    display: none;
+  }
+
+  .txtarea {
+    height: 150px;
+    width: 500px;
+  }
+
+  .icons3 {
+    margin-top: 100px;
+  }
+}
+
+@media screen and (max-width: 991px) {
```

Comment:
Media query to low

- [ ] Kindly select a value from `992 - 1100px` to target desktop screens